README.md

```diff
@@ -21,8 +21,9 @@ yarn electron:build
 
 ### Run your tests
 ```
-yarn run test
+yarn run test:unit
```

Comment:
semantics. test or tests?

Reply:
I think `test`. The project was created with [vue-cli](https://cli.vuejs.org/), which created the scripts that are available. I think I may have overwritten the README initially.